### ISSUE TITLE
Check that Chandra regions are being handled properly and masks can be created

### ISSUE THREAD

**Issue by DavidT3** [DavidT3/XGA#1245]
This won't be possible until I can get DAXA to make region files from Chandra images, but when it is you'll need to alter your configuration file to point at the region files in that test dataset (I'll let you know when they're sorted).

That should mean that XGA loads in the region files, tries to match to them based on the type of source being declared, and will also be able to exclude contaminating sources from photometric analyses/spectral generation.

Testing this should be simple - you can run the get_interloper_regions() method of a declared source (**not a BaseSource though** - try a GalaxyCluster) and it should return a set of contaminating regions if everything has been loaded properly.

**Comment by DavidT3** [comment]
This shouldn't be the immediate priority (that would be the phot-product-generating functions), but once that is done and I've made some region files for some of the data we have, we need to make sure the image masking is working properly. Then we'll turn our attention to being able to create the regions needed for spectral generation (that'll be another task though).

**Comment by DavidT3** [comment]
Certainly seems to be working, at least for the creation of image masks - @rz-wang is currently working on the generation of spectra, which necessitates further use of the regions read in by XGA sources, so that'll be the final test of whether everything is working properly.
